### ISSUE TITLE
Invalid default port mappings

### ISSUE THREAD

**Issue by Neamar** [D1ceWard/plausible_on_dokku#6]
I saw your commit on 50853d6db6365e0edd3f227b2013611f986abf6c and tried reinstalling from scratch.

For some reason, even on this new app, Dokku generated a default proxy port mapping:

```
-----> Port mappings for plausible
    -----> scheme  host port  container port
    http           80         5000
    https          443        5000
```

To get things working, I had to add this: `dokku proxy:ports-set plausible https:443:80` otherwise nginx was unable to connect to port 5000.

I checked other apps, and using port 5000 on the container seems to be the default, so maybe this line needs to be added to the README?

Thanks for your time!

**Comment by D1ceWard** [comment]
Mmmh, I had done some tests and Dokku managed to get the right port automatically with the EXPOSE instruction in the Dockerfile. With which version of Dokku this happened? Did the application already have the proxy configured by a previous deployment?

**Comment by Neamar** [comment]
I did an app:destroy before, so I believe I started from scratch completely!

running `dokku version 0.24.2`

**Comment by D1ceWard** [comment]
So, I re-tested with the latest version of Dokku (0.24.7) with a new fresh app and it seems to succeed in retrieving the port coming from the EXPOSE statement of the Dockerfile (see screen below), I will try to re-test with version 0.24.2 of Dokku and for Letsencrypt (443 mapping) when I have time.

![image](https://user-images.githubusercontent.com/8332823/117365224-45533680-aebf-11eb-92e9-f5e08b2ac66e.png)
![image](https://user-images.githubusercontent.com/8332823/117366207-b47d5a80-aec0-11eb-89b4-48f0f691ad5c.png)

**Comment by Neamar** [comment]
I must have messed up somewhere then, or have default config from other apps.

Thanks for investigating, I'll close, seems to be related to me

**Comment by D1ceWard** [comment]
I know it's been a while but I found the cause that prevents Dokku from auto-detecting the correct port. While doing another project (uptime_kuma_on_dokku) I noticed that configuring the domain before the first deployment defaults to port 5000 and prevents auto-detection of the port from the Dockerfile. So for now in the installation guide I have moved the domain configuration to the end, I will do more testing and maybe change the port in the container to port 5000 instead of 80.
